cmd/kubeadm/app/constants/constants.go

```diff
@@ -262,6 +262,10 @@ const (
 	// DefaultEtcdVersion indicates the default etcd version that kubeadm uses
 	DefaultEtcdVersion = "3.3.10"
```

Comment:
we forgot to change this to `3.3.15` (or `3.3.15-0`) in the other PR.

Reply:
I'll fix here.